web/src/locust/run.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Setting `set -eu` and `set -o pipefail` is a [good idea](http://gfxmonk.net/2012/06/17/my-new-bash-script-prelude.html) in most bash scripts.